app/helpers/formatPapersQuery.tsx

```diff
@@ -0,0 +1,47 @@
+interface IFormatPapersQueryParams {
```

Comment:
I strongly suggest to make class has methods for both parser and builder.

Reply:
I don't think so. I think to make functions a class is more difficult to manage code.